agent/util-scripts/gold/pbench-register-tool-trigger/test-35.txt

```diff
@@ -1,11 +1,16 @@
 +++ Running test-35 pbench-register-tool-trigger --group=default --start-trigger="START:DEFAULT" --stop-trigger="STOP DEFAULT"
 pbench-register-tool-trigger: the start trigger cannot have a colon in it: "START:DEFAULT"
+pbench-register-tool-trigger: the start trigger cannot have a colon in it: "START:DEFAULT"
 --- Finished test-35 pbench-register-tool-trigger (status=1)
 +++ pbench tree state
 /var/tmp/pbench-test-utils/pbench
+/var/tmp/pbench-test-utils/pbench/pbench.log
 /var/tmp/pbench-test-utils/pbench/tmp
 /var/tmp/pbench-test-utils/pbench/tools-v1-default
 /var/tmp/pbench-test-utils/pbench/tools-v1-default/testhost.example.com
 /var/tmp/pbench-test-utils/pbench/tools-v1-default/testhost.example.com/pidstat
 === /var/tmp/pbench-test-utils/pbench/tools-v1-default/testhost.example.com/pidstat:
 --- pbench tree state
++++ pbench.log file contents
+[ERROR][1900-01-01T00:00:00.000000] pbench-register-tool-trigger: the start trigger cannot have a colon in it: "START:DEFAULT"
```

Comment:
I hadn't thought this through reading the code; but this is actually a little weird, and I'm not sure we really need to be logging user input errors, especially at "ERROR" level.